ViewModel/ImpactAreaScenario/Editor/SpecificIASEditorVM.cs

```diff
@@ -460,42 +464,57 @@ private void PreviewCompute()
             SimulationCreator sc = new SimulationCreator(freqElem, inOutElem, ratElem, extIntElem, leveeElem,
                 stageDamageElem, CurrentImpactArea.ID);
 
-            FdaValidationResult configurationValidationResult = sc.IsConfigurationValid();
-            if(configurationValidationResult.IsValid)
+            int thresholdIndex = 1;
```

Comment:
I don't see a use for this thresholdIndex

Reply:
it is being used:
![image](https://user-images.githubusercontent.com/47757848/162526305-221cdf8a-0346-4b3d-b49e-fcaef7b19844.png)